### ISSUE TITLE
Filesystem does not support real SEEK_DATA/SEEK_HOLE

### ISSUE THREAD

**Issue by anjalirx-intel** [oscarlab/graphene#2537]
## Description of the problem
filesystem maybe use generic_file_llseek(), not support real SEEK_DATA/SEEK_HOLE

## Steps to reproduce
Run LTP Test lseek11

## Expected results
TINFO: The block size is 4096
TPASS: The 0th test passed: SEEK_DATA from startblock 0 offset 0
TPASS: The 1th test passed: SEEK_DATA from startblock 0 offset 4
TPASS: The 2th test passed: SEEK_HOLE from startblock 0 offset 0
TPASS: The 3th test passed: SEEK_HOLE from startblock 0 offset 4
TPASS: The 4th test passed: SEEK_HOLE from startblock 1 offset 0
TPASS: The 5th test passed: SEEK_HOLE from startblock 1 offset 128
TPASS: The 6th test passed: SEEK_DATA from startblock 1 offset 0
TPASS: The 7th test passed: SEEK_DATA from startblock 10 offset -1
TPASS: The 8th test passed: SEEK_DATA from startblock 10 offset 0
TPASS: The 9th test passed: SEEK_DATA from startblock 10 offset 4
TPASS: The 10th test passed: SEEK_HOLE from startblock 10 offset 0
TPASS: The 11th test passed: SEEK_HOLE from startblock 10 offset 4
TPASS: The 12th test passed: SEEK_HOLE from startblock 11 offset 128
TPASS: The 13th test passed: SEEK_DATA from startblock 11 offset 128
TPASS: The 14th test passed: SEEK_HOLE from startblock 30 offset -128

Summary:
passed   0
failed   0
skipped  0
warnings 0

## Actual results
TINFO: Timeout per run is 0h 05m 00s
TCONF: filesystem maybe use generic_file_llseek(), not support real SEEK_DATA/SEEK_HOLE

Summary:
passed   0
failed   0
skipped  1
warnings 0

## Label
Feature Request, not bug

**Comment by mkow** [comment]
> filesystem maybe use generic_file_llseek(), not support real SEEK_DATA/SEEK_HOLE

Sorry, I don't understand this sentence. Could you rephrase it?

**Comment by anjalirx-intel** [comment]
@mkow This is the statement from lseek11 log, overall it means Filesystem might be using generic_file_llseek(). But all this test needs support for SEEK_DATA/SEEK_HOLE

I don't know how much it will helps, but i could found little info on below link about generic_file_llseek

**Comment by dimakuv** [comment]
Graphene currently doesn't support `SEEK_DATA` and `SEEK_HOLE` and apparently no real-world applications request this feature. So let's consider it as "Linux feature not implemented in Graphene and not needed by real-world apps". And disable this particular LTP test.
